examples/text_summarization/pointer_summarizer/src/eval.py

```diff
@@ -0,0 +1,99 @@
+import os,sys,inspect
+sys.path.insert(0,'..')
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir)
+
+import time
+import tensorflow as tf
+import paddle
+
+from utils import config
+from utils.batcher import Batcher
+from utils.data import Vocab
+
+from utils.utils import calc_running_avg_loss
+from train_util import get_input_from_batch, get_output_from_batch
+from model import Model
+
+# only use certain GPUS
+use_cuda = config.use_gpu and paddle.is_compiled_with_cuda()
+if use_cuda:
+    paddle.set_device("gpu:0")
+print("Using cuda? ... ", use_cuda)
+
+
+class Evaluate(object):
```

Comment:
Evaluate is a verb, it seems not a proper way as a class name. How about Evaluator? Since we need to use Object-oriented programing.